loader.js

```diff
@@ -23,12 +23,7 @@ module.exports.pitch = function(request) {
 	var loaders = this.loaders.slice(this.loaderIndex + 1);
 	this.addDependency(this.resourcePath);
 	// We already in child compiler, return empty bundle
-	if(this[NS] === undefined) {
-		throw new Error(
-			'"extract-text-webpack-plugin" loader is used without the corresponding plugin, ' +
-			'refer to https://github.com/webpack/extract-text-webpack-plugin for the usage example'
-		);
-	} else if(this[NS] === false) {
+	if(this[NS] === undefined || this[NS] === false) {
 		return "";
```

Comment:
```
console.warn(`
  The extract-text-webpack-plugin loader is used without the corresponding plugin,
  in case this is unintentional refer to https://github.com/webpack/extract-text-webpack-plugin 
  for correct usage (example)
`)
```

Or similiar 😛